web_app/src/ArticleDisplay.js

```diff
@@ -30,6 +33,30 @@ class ArticleDisplay extends React.Component {
         }
     }
 
+    componentDidMount = () => {
```

Comment:
@SeeratDheer don't we get all the data for the article from the props itself? Then you don't need to fetch the details of the article again